docs/reference/contributing/target/bootloader.md

```diff
@@ -1,18 +1,17 @@
 ## Bootloader
 
-To enable firmware update capability via bootloader on a new target instructions as well as an example of how to do it can be found in the mbed OS 5 handbook.
+Arm Mbed OS allows you to update your firmware. Our Pelion IoT platform enables this capability. The following documentation includes instructions and examples on how to update firmware on a new target.
```

Comment:
1. ..update your "device" firmware.
2. Remove "new" in the last line as the target isn't new while updating the FW.
3. Add after ..this capability: "One of the dependencies is a bootloader running on the device capable of updating the device firmware. The following docs include..." 

Should be good to go with this.. thanks @AnotherButler

Reply:
Good catches